core-ui/src/components/Lambdas/LambdaDetails/Tabs/Code/LambdaVariables/InjectedVariables.js

```diff
@@ -2,30 +2,22 @@ import React from 'react';
 import LuigiClient from '@luigi-project/client';
 
 import { Icon, InfoLabel } from 'fundamental-react';
-import { GenericList, Tooltip } from 'react-shared';
+import { EMPTY_TEXT_PLACEHOLDER, GenericList, Tooltip } from 'react-shared';
 
 import {
   VARIABLE_VALIDATION,
   VARIABLE_TYPE,
   WARNINGS_VARIABLE_VALIDATION,
```

Comment:
Will those be moved in a separate task then?

Reply:
They are constant variables but not translations.